org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/XMLFoldings.java

```diff
@@ -49,6 +49,10 @@ public XMLFoldings(XMLExtensionsRegistry extensionsRegistry) {
 		this.extensionsRegistry = extensionsRegistry;
 	}
 
+	public boolean isIncludeClosingTagInFold(XMLFoldingSettings context) {

```

Comment:
As it is not an important method, I will move it after `getFoldingRanges` which is main entry method. More this method is used only in this class, so you can set to private. At the end, this method don't need some fields of the XMLFoldings, so I will add static, in other words : 

`private static isIncludeClosingTagInFold(XMLFoldingSettings foldingSettings)`